### ISSUE TITLE
Update support of VS2017

### ISSUE THREAD

**Issue by 3F** [3F/vsSolutionBuildEvent#39]
Continuation of #36
------

**Any pull requests are welcome to support: VS2010, VS2012, VS2013, VS2015, VS2017**

[v0.12.8+](http://vssbe.r-eg.net/Changelist/#vsix), obsolete:

```
NEW: Started support of Visual Studio 15 /tested on Enterprise Preview 4 (25618.00)
```
**[New replies](https://developercommunity.visualstudio.com/content/problem/1180/index.html):**

>> More details about the changes to VS extensibility for VS 2017 can be found here:
>> 
>> https://blogs.msdn.microsoft.com/visualstudio/2016/11/10/extensibility-in-visual-studio-15-increasing-reliability-and-performance/
>> 
>> Technical details on the extensibility changes for VS 2017 can be found here:
>> 
>> https://docs.microsoft.com/en-us/visualstudio/extensibility/what-s-new-in-the-visual-studio-2017-sdk
>> 
>> Thanks,
>> 
>> - The Visual Studio Team

---------

>> More details about the changes to VS extensibility for VS 2017 can be found here:
>
>:) I see my extension in your screenshot
>
>
>Good ! but trivial question:
>
>How to distribute my packages for older VS which understand only VSIXv1 ?
>I mean: "minimal install will contain just the core editor" - really cool, but how to continue support all VS versions [which already supports this] perfectly ?
>
>well, I still need answer/suggestions:
>
>> should I finally stop support of VS2010 or provide 2 or 3 variant of my package for each version of Visual Studio ?!
>> or there are more ways ?
>
>Seems I need configure project to compile this for several versions and then create new pages in your new marketplace for all this, with new vsix ID -_-
>
>here https://docs.microsoft.com/en-us/visualstudio/extensibility/how-to-migrate-extensibility-projects-to-visual-studio-2017
>I see new doc. for how to upgrade from VSIX v2 to VSIX v3, later I will try it. Thanks.

**Comment by s00ther** [comment]
Hi, 
do you have any update on the compatibility with VS 2017?
It would be great to have your extension again :)

**Comment by 3F** [comment]
@s00ther 

**v0.12.10.10901** is still **compatible with VS2017**, except installation manifest. That is, temporarily, click [ok] buttons for any warnings when you install it.

Also [CIM](https://www.nuget.org/packages/vsSBE.CI.MSBuild/) version still have no any problems for build servers and for local usage.

**Please don't worry,** I've plan support this product because it's actively used for my projects at least: https://github.com/3F?tab=repositories

**However,** currently I have no time/priority for this. As soon as possible...
Anyone else may help to update this support for VS2017 (VS2010+). In general, as I said above it affects only for manifest of installation but the main problem of this you can read in #36 & #39 issues. So, any elegant way to support all installation from VS2010 are very welcome.

**Comment by s00ther** [comment]
@3F Thank you very much
That's a really good news

**Comment by 3F** [comment]
Closed due the new Issue: #44. Please continue any related discussions there.
